packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -1,54 +1,64 @@
 import React, { FC, ReactNode } from 'react';
-import classNames from 'classnames';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
 import { Icon } from '../Icon/Icon';
 import { IconName } from '../../types/icon';
 
 export type AlertVariant = 'success' | 'warning' | 'error' | 'info';
 
-interface AlertProps {
+export interface Props {
   title: string;
-  buttonText?: string;
-  onButtonClick?: (event: React.MouseEvent) => void;
   onRemove?: (event: React.MouseEvent) => void;
   severity?: AlertVariant;
   children?: ReactNode;
+  removeButtonContent?: ReactNode | string;
```

Comment:
I can't find any usages of this property, maybe we can just remove it? Maybe it's used in enterprise? TBH I don't think we should allow customization of the close/remove button.

Reply:
I added `removeButtonContent` as an alternative to `buttonText`, to unify both button and remove properties into one. Apparently there are some cases where a button is needed instead of the close icon.